These projects are not needed to run tests and/or we do not have them packaged.

--- uvicorn-0.31.1/requirements.txt.orig
+++ uvicorn-0.31.1/requirements.txt
@@ -12,20 +12,12 @@
 twine==5.1.1
 
 # Testing
-ruff==0.6.8
 pytest==8.3.3
 pytest-mock==3.14.0
 mypy==1.11.2
-types-click==7.1.8
 types-pyyaml==6.0.12.20240917
 trustme==1.1.0
 cryptography==43.0.1
 coverage==7.6.1
-coverage-conditional-plugin==0.9.0
 httpx==0.27.2
-watchgod==0.8.2
 anyio==3.7.1
-
-# Documentation
-mkdocs==1.6.1
-mkdocs-material==9.5.39
